components/FlowCard.js

```diff
@@ -86,7 +86,16 @@ const StyledButton = styled.button`
 const StyledButtonWrapper = styled.button`
   all: unset;
   position: absolute;
+  width: 1.5rem;
+  height: 1.5rem;
+  border-radius: 50%;
```

Comment:
why is here a `border-radius`?

Reply:
Totally useless 😅 I first had a round background and forgot to delete the border-radius. Thanks for the hint